CMP1903_A1_2324/Program.cs

```diff
@@ -10,10 +11,44 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            /*
-             * Create a Game object and call its methods.
-             * Create a Testing object to verify the output and operation of the other classes.
-             */
+            Console.WriteLine("How many times would you like to roll the dice");
+            string NumTimes = Console.ReadLine();
```

Comment:
What happens if 0 is input in the console, or a space? Do these get caught by the try; catch; statement?

Reply:
It doesn't roll any